pkg/ebpf/common.go

```diff
@@ -78,27 +79,52 @@ func verifyOSVersion(kernelCode uint32, platform string, exclusionList []string)
 		return true, nil
 	}
 
-	if isCentOS(platform) && kernelCode >= centOSMinRequiredKernelCode {
-		return true, nil
-	}
-
 	if isUbuntu(platform) {
 		if kernelCode >= linuxKernelVersionCode(4, 4, 119) && kernelCode <= linuxKernelVersionCode(4, 4, 126) {
 			return false, fmt.Errorf("got ubuntu kernel %s with known bug on platform: %s, see: https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1763454", kernelCodeToString(kernelCode), platform)
 		}
 	}
 
-	if kernelCode < minRequiredKernelCode {
-		return false, fmt.Errorf(
-			"incompatible linux version. at least %s (%d) required, got %s (%d)",
-			kernelCodeToString(minRequiredKernelCode),
-			minRequiredKernelCode,
-			kernelCodeToString(kernelCode),
-			kernelCode,
-		)
+	kallsyms, err := readKallsyms()
+	if err != nil {
+		log.Warnf("error reading /proc/kallsyms file: %s", err)
```

Comment:
I guess this warning message is not totally accurate because the procRoot is not always `/proc/kallsyms` right?

Reply:
I guess warning on the return error should enough right?

Reply:
👍